gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -30,15 +29,15 @@
 from .store import IPLD
 
 
-class Creation(Convenience):
+class Transform(Convenience):
     """
     Base class for transforming a collection of downloaded input files in NetCDF4 Classic format into
     (sequentially) kerchunk JSONs, a MultiZarr Kerchunk JSON, and finally an Xarray Dataset based on that MultiZarr.
     """
 
     # KERCHUNKING
 
-    def create_zarr_json(self, force_overwrite: bool = True):
+    def create_zarr_json(self, force_overwrite: bool = True, file_filter: str = None, outfile_path: str = None):
```

Comment:
type annotation should be `Optional[str]`

Reply:
I kind of prefer `str|None`. Cleaner looking and less to type. Thoughts?

Reply:
Sure, `str | None` is good